requirements.txt

```diff
@@ -1,9 +1,9 @@
-Cython
-Owlready2==0.30
-graphviz
-PyYAML
-blessings
-Pygments
-rdflib
-semver
-pydot
+Cython~=0.29.21
+Owlready2~=0.28
+graphviz~=0.16
+PyYAML~=5.4.0
+blessings~=1.7
+Pygments~=2.8.0
+rdflib~=5.0.0
+semver~=2.13.0
+pydot~=1.4.2
```

Comment:
Should be removed together with the deprecated emmo/ontograph.py
We should make a new issue for that.